react/components/AddBlock/index.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import Styles from 'react/styles';
+import { Textarea } from 'react/components/UI/GenericInput';
+import { isURL } from 'validator';
+
+const Outline = styled.div`
+  width: ${Styles.Constants.blockWidth};
+  height: ${Styles.Constants.blockWidth};
+  background-color: ${Styles.Colors.gray.semiLight};
+  border-radius: .25em;
+  padding: 5px;
+  box-sizing: border-box;
+`;
```

Comment:
We got a mixin for this. The trick is that done this way it'll mess up alignment. The mixin just avoids this issue by mimicking the outline property.

https://github.com/aredotna/ervell/blob/64fc8b4005f5ad68cbc96d01fed5869fd5fbcd36/react/styles/Mixins.js#L5-L17

Reply:
Ah got it.